partner_report_open_invoices/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+============================
+Partner Report Open Invoices
+============================
+
+This module will create a report that contains the details of open invoices for partner in partner form view.
+
+
+Installation
+============
+
+This module depends on:
+
+* account
+
+Configuration
+=============
+
+There is nothing to configure.
+
+Known issues / Roadmap
+======================
+
+ * No known issues.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/account-financial-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/account-financial-tools/issues/new?body=module:%20partner_report_open_invoices_mode%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
Please the updated version of Bug Tracker section, see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#bug-tracker as reference.